hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemAuthorization.java

```diff
@@ -327,7 +328,15 @@ private void executeOp(Path reqPath, AzureBlobFileSystem fs,
       fs.open(reqPath);
       break;
     case Open:
-      fs.open(reqPath);
+      try(InputStream is = fs.open(reqPath)) {
```

Comment:
Since this test has nothing to do with head optimization, don't really want to override Configuration and set the optimization to false. But I am open to take the change if it brings better code readability, but having this makes a testcase in the codebase to check SAS authentication for read flow. What you feel?